editor/converter.cpp

```diff
@@ -0,0 +1,3667 @@
+	//	// {"extends","size"}, // BoxShape3D, LightmapGI broke ReflectionProbe
+	//	// {"meta","meta_pressed"},// This may broke a lot of comments and user variables
+	//	// {"pause_mode","process_mode"}, // Node - Cyclic rename, look for others
+	//	// {"rotate","rotates"}, // PathFollow2D - probably function exists with same name
+	//	// {"shift","shift_pressed"},// This may broke a lot of comments and user variables
+	//	{ "as_normalmap", "as_normal_map" }, // NoiseTexture
+	//	{ "autowrap", "autowrap_mode" }, // Label
+	//	{ "cast_to", "target_position" }, // RayCast2D, RayCast3D
+	//	{ "doubleclick", "double_click" }, // InputEventMouseButton
+	//	{ "group", "button_group" }, // BaseButton
+	//	{ "process_mode", "process_callback" }, // AnimationTree, Camera2D
+	//	{ "scancode", "keycode" }, // InputEventKey
+	//	{ "toplevel", "top_level" }, // Node
+	//	{ "window_title", "title" }, // Window
+	//	{ "wrap_enabled", "wrap_mode" }, // TextEdit
+	//	{ "zfar", "far" }, // Camera3D
+	//	{ "znear", "near" }, // Camera3D
+	{ "bbcode_text", "text" }, // RichTextLabel
+	{ "caret_moving_by_right_click", "caret_move_on_right_click" }, // TextEdit
+	{ "caret_position", "caret_column" }, // LineEdit
+	{ "drag_margin_bottom", "drag_bottom_margin" }, // Camera2D
+	{ "drag_margin_h_enabled", "drag_horizontal_enabled" }, // Camera2D
+	{ "drag_margin_left", "drag_left_margin" }, // Camera2D
+	{ "drag_margin_right", "drag_right_margin" }, // Camera2D
+	{ "drag_margin_top", "drag_top_margin" }, // Camera2D
+	{ "drag_margin_v_enabled", "drag_vertical_enabled" }, // Camera2D
+	{ "enabled_focus_mode", "focus_mode" }, // BaseButton - Removed
+	{ "extra_spacing_bottom", "spacing_bottom" }, // Font
+	{ "extra_spacing_top", "spacing_top" }, // Font
+	{ "focus_neighbour_bottom", "focus_neighbor_bottom" }, // Control
+	{ "focus_neighbour_left", "focus_neighbor_left" }, // Control
+	{ "focus_neighbour_right", "focus_neighbor_right" }, // Control
+	{ "focus_neighbour_top", "focus_neighbor_top" }, // Control
+	{ "global_rate_scale", "playback_speed_scale" }, // AudioServer
+	{ "gravity_distance_scale", "gravity_point_distance_scale" }, // Area2D
+	{ "gravity_vec", "gravity_direction" }, // Area2D
+	{ "iterations_per_second", "physics_ticks_per_second" }, // Engine
+	{ "margin_bottom", "offset_bottom" }, // Control broke NinePatchRect, StyleBox
+	{ "margin_left", "offset_left" }, // Control broke NinePatchRect, StyleBox
+	{ "margin_right", "offset_right" }, // Control broke NinePatchRect, StyleBox
+	{ "margin_top", "offset_top" }, // Control broke NinePatchRect, StyleBox
+	{ "offset_h", "drag_horizontal_offset" }, // Camera2D
+	{ "offset_v", "drag_vertical_offset" }, // Camera2D
+	{ "physical_scancode", "physical_keycode" }, // InputEventKey
+	{ "popup_exclusive", "exclusive" }, // Window
+	{ "region_filter_clip", "region_filter_clip_enabled" }, // Sprite2D
+	{ "size_override_stretch", "size_2d_override_stretch" }, // SubViewport
```

Comment:
Environment SSR properties were renamed in https://github.com/godotengine/godot/pull/59526

```suggestion
	{ "ss_reflections_enabled", "ssr_enabled" }, // Environment
	{ "ss_reflections_max_steps", "ssr_max_steps" }, // Environment
	{ "ss_reflections_fade_in", "ssr_fade_in" }, // Environment
	{ "ss_reflections_fade_out", "ssr_fade_out" }, // Environment
	{ "ss_reflections_depth_tolerance", "ssr_depth_tolerance" }, // Environment
```
Ignore the line removal in the suggestion, I can't figure out how to suggest a change without removing a line :/